3.1/ReleaseNotes.html

```diff
@@ -10,6 +10,43 @@
   <body>
     <div class="container">
     <h1>Proclaim Version 3.1</h1>
+    <div data-version="3.1.0.0064">
+      <h3>3.1.0.0064 Monday, May 24, 2021</h3>
+      <ul>
+        <li data-case="PROC-9406">Add new canvas sizes for ultrawide (21:9) and double/triple variants of 16:9 and 4:3.</li>
+        <li data-case="PROC-9365">Use yellow or green for chords in the confidence monitor.</li>
+        <li data-case="PROC-9593">Honor font spacing setting for Chords/Lyrics layout on Confidence monitor.</li>
+        <li data-case="PROC-9526">Cleanup suggested presentation logic for multiple presentations the same day.</li>
+
+        <li>Revamp Faithlife Music sync
+            <ul>
+            <li>Faster sync</li>
+            <li>Better conflict resolution</li>
+            <li>Store all arrangements locally for offline song creation and edits, sync later when back on the internet.</li>
+            </ul>
+        </li>
+        <li data-case="PROC-9552">Ensure Proclaim and Worship team visibility and permissions are created appropriately on Faithlife.com.</li>
+        <li data-case="PROC-9532">Reduce chances of linking imported song lyrics to the incorrect song on Faithlife Music.</li>
+        <li data-case="PROC-9467">Don't allow old song lyrics data to override the newest revision reusing old service items.</li>
+        <li data-case="PROC-9392">Allow user to specify a worship team during song import.</li>
+
+        <li data-case="PROC-9524">Fix rare crash that can occur when transitioning slides while alt-content is still generating.</li>
+        <li data-case="PROC-9501">Fix multi-select bug that sets all foreground images to the selected item's foreground image.</li>
+        <li data-case="PROC-9447">Allow deleting all textFields on alt content (with the exception being the lyrics field).</li>
+        <li data-case="PROC-9443">Importing from PPT or PCO leaves service item editable title in unselected edit state</li>
+        <li data-case="PROC-9426">Ensure recently deleted presentations don't continue to show in the open menu list.</li>
+        <li data-case="PROC-9358">Fix UI flash when adding items to image slideshow.</li>
+        <li data-case="PROC-9301">Prevent relaunch banner from displaying multiple times.</li>
+      </ul>
+      <h3>Mac only</h3>
+      <ul>
+        <li data-case="PROC-9353">Fix bug that could cause subsequent imports of the same Keynote/PowerPoint to be missing recent changes.</li>
+        <li data-case="PROC-9514">Fallback to Fade when using Circle Reveal transition to avoid many issues relating to blank slides or incorrect content showing On Air on Big Sur.</li>
```

Comment:
```suggestion
        <li data-case="PROC-9514">Fall back to Fade transition when using Circle Reveal to avoid issues with  blank slides or incorrect content while On Air.</li>
```